app/config/config.exs

```diff
@@ -7,7 +7,9 @@ use Mix.Config
 
 # General application configuration
 config :rocdev,
-  ecto_repos: [Rocdev.Repo]
+  ecto_repos: [Rocdev.Repo],
+  slack_invite_base_url: System.get_env("SLACK_INVITE_API_BASE") || "${SLACK_INVITE_API_BASE}",
+  slack_invite_api_token: System.get_env("SLACK_INVITE_API_TOKEN") || "${SLACK_INVITE_API_TOKEN}"
```

Comment:
What's the point of the `"${....}"` versions?

Reply:
I read somewhere they were interpolated at runtime. I haven't validated it at all nor have I thought through how I'll deploy this thing.

Reply:
Nope, not in general anyways.  If it's done by the Slack lib you may be ok?

One idiom is to use `{:system, "NAME_OF_VAR"}` and interpret that at runtime, though it's been seen more as a thing that should be left up to the application developer.  Ecto used to do this with `{:system, _}` automatically but it was removed for Reasons, and they now recommend for the developer to do it in their own application.